### PR TITLE
starting fpl tests

### PR DESCRIPTION
**Pull request by clewis7**
should be able to starting creating ground truths 

works on #152

### CODE REVIEW

**Comment by clewis7** [comment]
current updates: 

creating ground truth for simple and comparison seems to be working 

commented out some of the stuff that pygfx uses (can try to integrate back in but not sure if it is relevant for us),  can do zenodo upload eventually for ground truth files like for mesmerize

going to continue adding examples

File: examples/features/2Dcmap.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+plot = Plot(canvas=canvas, renderer=renderer)
+
+data = np.load("../data/2Drand.npy")
+
+# plot the image data
+image_graphic = plot.add_image(data=data, name="random-image")
+
+plot.show()
+
+image_graphic.cmap = "viridis"
```
Comment (kushalkolar) [inline examples/features/2Dcmap.py:27]: test a few more cmaps, "jet", "gray", "turbo"

File: examples/features/2Dcmap.py
Comment (kushalkolar) [inline examples/features/2Dcmap.py:1]: don't start names with a number and use all lower case

File: examples/features/colorslice.py
Comment (kushalkolar) [inline examples/features/colorslice.py:1]: put line, image, etc. graphic tests in individual dirs

File: examples/validation/2Drgb.py
Comment (kushalkolar) [inline examples/validation/2Drgb.py:1]: organize the tests by graphic type

File: notebooks/gridplot.ipynb
Comment (kushalkolar) [inline notebooks/gridplot.ipynb:1]: remove these from here

**Review by kushalkolar — CHANGES_REQUESTED** [review]

File: examples/features/2Drgbvminvmax.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+# plot the image data
+image_graphic = plot.add_image(data=im, name="iio astronaut")
+
+plot.show()
+
+image_graphic.vmin = 0.5
+image_graphic.vmax = 0.75
+
+img = np.asarray(plot.renderer.target.draw())
```
Comment (kushalkolar) [inline examples/features/2Drgbvminvmax.py:31]: take images before and after changing things

File: examples/features/colorslice.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+# indexing to assign colormaps to entire lines or segments
+sinc_graphic.cmap[10:50] = "gray"
+sine_graphic.cmap = "seismic"
+
+# more complex indexing, set the blue value directly from an array
+cosine_graphic.colors[65:90, 0] = np.linspace(0, 1, 90-65)
+
+plot.center_scene()
+
+img = np.asarray(plot.renderer.target.draw())
```
Comment (kushalkolar) [inline examples/features/colorslice.py:62]: take images before and after changing things

**Comment by kushalkolar** [comment]
a good start but you'll have to start a new branch and move over all the tests without the large binary files

**Pull request closed without merging**